game/view/helpers/card.lua

```diff
@@ -101,7 +101,8 @@ function CARD.drawInfo(card, x, y, width, alpha)
   g.translate(0, _title_font:getHeight())
 
   _text_font.set()
-  g.printf(card:getDescription() or "[No description]", 0, 0, width)
+  local desc = card:getDescription():gsub("[^\n][\n][^\n]", " ")
```

Comment:
This doesn't work in Fireball's description: it eats the characters before and after the breakline.